pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -88,9 +88,10 @@ let
       throw "Please add a qtPlatformCross entry for ${plat.config}";
 
   # Per https://doc.qt.io/qt-5/macos.html#supported-versions: build SDK = 13.x or 14.x.
-  # SDK 13.x causes weird linking errors on x86_64-darwin, so use 14.x
+  # Despite advertising support for the macOS 14 SDK, the build system sets the maximum to 13 and complains
+  # about 14, so we just use that.
```

Comment:
The build systems seems to be OK with 14 now, so maybe we should just keep that?

Reply:
Okay, adjusted to just drop the comment. I’m verifying if the build goes through but obviously it’s very slow going.